t/129-ssl-socket.t

```diff
@@ -2618,3 +2676,410 @@ qr/\[error\] .* ngx.socket sslhandshake: expecting 1 ~ 5 arguments \(including t
 --- no_error_log
 [alert]
 --- timeout: 5
+
+
+
+=== TEST 33: setsslcert, too many arguments
```

Comment:
These new tests should go to a separate test file. This `129-ssl-socket.t` file is already quite large.